action.yml

```diff
@@ -30,15 +34,19 @@ runs:
         git config --global user.name "$(git log -1 --pretty=format:'%an')"
         git remote set-url origin "https://x-access-token:${{ github.token }}@github.com/$GITHUB_REPOSITORY"
       shell: bash
+      working-directory: ${{ inputs.working-directory }}
```

Comment:
Is this useful? The repo folder or sub folders will not affect these commands.

If it's submodule in a repo folder, this will not help. We can remove this.